### ISSUE TITLE
The print button prints a blank page.

### ISSUE THREAD

**Issue by rjpruitt16** [wonism/gatsby-advanced-blog#11]
I tried fixing the bug. I thought the issue was javascript related, so I tried downloading the react-to-print library. I tried with the dangerouslySet div with no luck, and then I tried with a regular div with the same bug. I keep getting the warning  <Provider> does not support changing `store` on the fly. 

In resume/index.jxs, 

```
import ReactToPrint from "react-to-print";

render() {
    const { data, printPage } = this.props;
    const resume = get('markdownRemark')(data);
    **const printButton = (
      <button type="button">
        <FaPinrt />
        Print
      </button>
    );**
    return (
      <Wrapper>
        <Clearfix>
          <Helmet>
            <title>
              WONISM | RESUME
            </title>
            <meta name="og:title" content="WONISM | RESUME" />
          </Helmet>
          <Clearfix>
            **<ReactToPrint
              trigger={() => <a href="#">Print this out!</a>}
              content={() => this.ref}
            />
            {printButton}**
          </Clearfix>
          <BasicInformation>
            <img
              src={profileUrl.default}
              alt=""
              width="120"
              height="120"
            />
            <h1>
              wonism
            </h1>
            <p>
              yocee57@gmail.com
            </p>
          </BasicInformation>
          <SocialInformation>
            <a
              href="https://github.com/wonism"
              target="_blank"
              rel="noreferrer noopener"
            >
              <FaGithub />
            </a>
            <a
              href="https://twitter.com/j1ism"
              target="_blank"
              rel="noreferrer noopener"
            >
              <FaTwitter />
            </a>
            <a
              href="https://www.facebook.com/j1.chic"
              target="_blank"
              rel="noreferrer noopener"
            >
              <FaFacebook />
            </a>
            <a
              href="https://www.linkedin.com/in/wonism/"
              target="_blank"
              rel="noreferrer noopener"
            >
              <FaLinkedin />
            </a>
          </SocialInformation>
          <MDInformation>
            <div
              id="Text"
              ref={(mdWrapper) => { this.$mdWrapper = mdWrapper; }}
              dangerouslySetInnerHTML={{ __html: get('html')(resume) }}
            />

            **<div ref={(mdWrapper) => { this.ref = mdWrapper; }}>
              <p> Non-dangerouslySetHTML</p>
            </div>**
          </MDInformation>
        </Clearfix>
      </Wrapper>
    );
  }
}
```

**Comment by wonism** [comment]
Ok, I will take a look as soon as possible.
Thanks for reporting this.

**Comment by wonism** [comment]
Fixed with 75ab83474b070b4b193cd335da7e7008dd09b733
